### ISSUE TITLE
Comment "ReSharper disable once" does not suppress warnings

### ISSUE THREAD

**Issue by IvanKonov** [CSharpAnalyzers/ExceptionalReSharper#58]
Comment "ReSharper disable once" does not suppress warnings
![default](https://user-images.githubusercontent.com/20556427/27987826-f610c240-641c-11e7-969f-656294c74671.PNG)

**Comment by kirbatious** [comment]
Yeah, I was excited to see Exceptional back in Resharper Extensions, but none of the Resharper Disable commands work for me now. It makes it fairly unusable as I want to be able to disable the warnings in unit test classes (as well as false positives in normal code), e.g.:
![image](https://user-images.githubusercontent.com/7012396/28166976-d41a3d98-67d1-11e7-984f-6e7e60fca89d.png)

**Comment by pedromagueija** [comment]
Hi guys,

Sorry about that. We recently had a merge of projects and one of the compilation symbols that fixed this issue was lost.

I'll fix it promptly.

**Comment by kirbatious** [comment]
Hi @pedromagueija - I just tried reinstalling Exceptional today and I still have the same issue. Is there a delay before extensions are available on https://resharper-plugins.jetbrains.com/api/v2/curated-feeds/Wave_v8.0/ ? Thanks again for looking into it!

**Comment by pedromagueija** [comment]
Hi @kirbatious,
I don't think it was released yet. But I do have a package myself if you want to test it,
Just bear in mind that when the release is done you'd have to uninstall "my" package and install the official one.

**Comment by RSuter** [comment]
The extension has to be published first... @pedromagueija is the fix already in the repo?

**Comment by pedromagueija** [comment]
@RSuter yep, it was on f8316c5. Will you do the release?

**Comment by RSuter** [comment]
Do you think we should merge this too? https://github.com/CSharpAnalyzers/ExceptionalReSharper/pull/61

**Comment by pedromagueija** [comment]
I haven't tested it myself. Have you had a chance to test it? If not, I could test it later today and give you feedback. After that, we could make the release.

What do you think?

**Comment by RSuter** [comment]
Im just trying to test it - have to upgrade R# first on my current machine... If everything looks fine, ill release...

**Comment by pedromagueija** [comment]
Great :)

**Comment by RSuter** [comment]
Ok it works, but sometimes only after i change something in the document (ie the analyzers is run again). If I just add the comment, it does not work directly - change something or reopen document then its fine. Do you have the same problems?

**Comment by RSuter** [comment]
v0.10.0.0 is released

**Comment by pedromagueija** [comment]
@RSuter I'm not experiencing that issue. As soon as I added the comment it warning went away. Is it with a specific comment only? I used // ReSharper disable once ExceptionNotDocumented

**Comment by RSuter** [comment]
Adding works for me too, but removing it and then undo breaks...

**Comment by pedromagueija** [comment]
Just tested the as you described:

- added comment (warning off)
- removed comment (warning on)
- Undo to revert remove comment (warning off)

Then did the same steps saving the file in between each operation.
Can someone else confirm the issue?

**Comment by pedromagueija** [comment]
Reproduced by pasting the comment on the same line as the exception throwing and then hitting return.
It only removes the warning after saving the file.

**Comment by RSuter** [comment]
Is this some sort of R# bug?

**Comment by pedromagueija** [comment]
I tried the same for a standard Inspection and it works. I think we should create a new issue and try to solve it there.

**Comment by kirbatious** [comment]
Wow, thanks guys! I just got 0.10.0.0 and it works! :tada:
